src/Raven.Server/Commercial/LicenseManager.cs

```diff
@@ -429,7 +429,7 @@ private void AssertCanAssignCores(int assignedCores)
             }
         }
 
-        public async Task Activate(License license, bool skipLeaseLicense, bool ensureNotPassive = true, bool forceActivate = false)
+        public async Task Activate(License license, bool skipLeaseLicense, string guid, bool ensureNotPassive = true, bool forceActivate = false)
```

Comment:
`guid` of what? It looks like this related to the license id. Better to name it properly